pkg/daemon/ceph/client/filesystem.go

```diff
@@ -93,6 +96,19 @@ func ListFilesystems(context *clusterd.Context, clusterName string) ([]CephFiles
 	return filesystems, nil
 }
 
+func GenerateMetadataPoolName(fsName string, metadataPool *model.Pool) *model.Pool {
+	metadataPool.Name = fmt.Sprintf("%s-%s", fsName, metaDataPoolSuffix)
+	return metadataPool
+
+}
+
+func GenerateDataPoolName(fsName string, dataPools []*model.Pool) []*model.Pool {
```

Comment:
ditto

Reply:
This should have a fuller description as well. In my words, this is a helper method that generates a new name for each pool in the input list suffixed with an appropriate index number, and it returns the renamed pool list. It also should end in an `s` (`GenerateDataPoolNames`) because it can take an array of inputs, not just a single pool.